### PR TITLE
Feature/hocs 3559 update foi topics

### PR DESCRIPTION
**Pull request by abarker66**
The FOI Topics list now uses the full active DCU topics list via a type-ahead component.

### CODE REVIEW

**Review by LaurieBennett — CHANGES_REQUESTED** [review]
One logger to remove possibly

File: server/lists/adapters/topicsfoi.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const byLabel = (a, b) => a.label.toUpperCase().localeCompare(b.label.toUpperCase());
+
+module.exports = async (data, { logger }) => {
+    logger.debug('FOI_TOPICS', { topics: data.length });
```
[the remark this answers is not in the document]
Comment (abarker66) [reply server/lists/adapters/topicsfoi.js:4]: Laurie, this is intentional - all/most the adapters log the data length to show the list size.

**Review by gary-newbolt — APPROVED** [review]
lgtm

File: server/lists/adapters/topicsfoi.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (LaurieBennett) [reply server/lists/adapters/topicsfoi.js:4]: Ah, that's fine then

**Review by LaurieBennett — APPROVED** [review]
All looks fine, pass

**Pull request merged**